src/helpers/email/send.ts

```diff
@@ -71,26 +71,28 @@ export async function newRequest(requestObj: any): Promise<any> {
 			if (!manager) {
 				return false;
 			} else if (manager.emails) {
-				let msg = requestTemplate(
-					requestObj.quantity,
-					requestingUser.name,
-					manager.name,
-					item.name,
-					store.name,
-					store._id,
-				);
-				let email = {
-					from: `ETHS Kitcoin Team <${email_from}>`,
-					replyTo: email_replyto,
-					to: manager.email,
-					subject:
-						`${requestingUser.name} requested ` +
-						(requestObj.quantity > 1 ? 'items!' : 'an item!'),
-					html: msg,
-				};
-				await transporter.sendMail(email).catch(console.log);
+				emails.push(manager.email!);
 			}
 		}
+		let msg: string = requestTemplate(
+			requestObj.quantity,
+			requestingUser.name,
+			'Store Manager',
+			item.name,
+			store.name,
+			store._id,
+		);
+		let email: Object = {
```

Comment:
Another typescript complaint: `Object` doesn't tell us anything that we (or typescript) didn't already know, so you might as well delete it. (or change it to `Mail.Options` for bonus points!)